chore: Update contributing docs (#9456)
getsentry/sentry-docs: docs/contributing/pages/components.mdx

```
````

> You can call out specific lines in a code block using the `{fromLineA-toLineB}` for ranges
> or `{lineA,lineB}` for specific lines (or a combination of both):

Reviewer: ```suggestion
You can bring attention to specific lines in a code block using the `{fromLineA-toLineB}` for ranges,
or `{lineA,lineB}` for specific lines (or a combination of the two):
```

```javascript {2} {tabTitle:Example}
function foo() {
  let lookat = "me";
  return "bar";
}
```

````markdown {tabTitle:Source}
```javascript {2}
function foo() {
  let lookat = "me";
  return "bar";
}
```
````

> Highlighting diffs is also supported using the `diff` language:

Reviewer: ```suggestion
You can also highlight diffs using the `diff` language:
```

````markdown {tabTitle:Example}
```diff
- minus one
+ plus one
```
````

> Or if you want to preserve syntax highlighting, you can add a `diff` metadata to any code block,

Reviewer: ```suggestion
If you want to preserve syntax highlighting, you can add `diff` metadata to any code block
```
